src/LogFormatter.php

```
@@ -449,6 +469,8 @@ public function setShowDurations($showDurations) {
      * @param string $str The string to write.
      */
     public function write($str) {
-        echo $str;
+        if (!feof($this->outputHandle)) {
+            fwrite($this->outputHandle, $str);
+        }
```

This silent failure could be hard to debug. Isn't a noisy error better?